spec/javascripts/cellSpec.js

```diff
@@ -0,0 +1,103 @@
+describe('Cell', function(){
+  var testCell; 
+
+  beforeEach(function(){
+    testCell = new Cell(0,0); 
+  });
+
+  describe('Should initalize', function(){
```

Comment:
I'm not crazy about tests like the ones in this describe block. They provide very little value, and they're something that needs to be maintained. I say stick to writing tests that assert on `behavior` not on `code`.